content/en/docs/azure/authentication-oidc.md

```diff
@@ -0,0 +1,209 @@
+        istio: ingressgateway
+    servers:
+    - hosts:
+        - '*'
+        port:
+        name: http
+        number: 80
+        protocol: HTTP
+              # Upgrade HTTP to HTTPS
+        tls:
+        httpsRedirect: true
+    - hosts:
+        - '*'
+        port:
+        name: https
+        number: 443
+        protocol: HTTPS
+        tls:
+        mode: SIMPLE
+        privateKey: /etc/istio/ingressgateway-certs/tls.key
+        serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
+    {{< /highlight >}}
+
+1. Expose Kubeflow with a loadBalancer service:
+
+    To expose Kubeflow with a LoadBalancer service, just change the type of the `istio-ingressgateway` service to `LoadBalancer`.
+
+    ```
+    kubectl patch service -n istio-system istio-ingressgateway -p '{"spec": {"type": "LoadBalancer"}}'
+    ```
+
+   After that, get the LoadBalancer's IP or Hostname from its status and create the necessary certificate.
+
+    ```
+    kubectl get svc -n istio-system istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0]}'
+    ```
+
+1. Create the Certificate with cert-manager:
+
+    Create a new `certficate.yaml` with below yaml to create a self-signed Certificate with the cert-manager. For production environment please use appropriate trusted CA Certificate.
+
+    {{< highlight yaml >}}
+    apiVersion: cert-manager.io/v1alpha2
+    kind: Certificate
+    metadata:
+    name: istio-ingressgateway-certs
+    namespace: istio-system
+    spec:
+    commonName: istio-ingressgateway.istio-system.svc
+        # Use ipAddresses if your LoadBalancer issues an IP
+    ipAddresses:
+    - <LoadBalancer IP>
+        # Use dnsNames if your LoadBalancer issues a hostname
+    dnsNames:
+    - <LoadBalancer HostName>
+    isCA: true
+    issuerRef:
+        kind: ClusterIssuer
+        name: kubeflow-self-signing-issuer
+    secretName: istio-ingressgateway-certs
+    {{< /highlight >}}
+
+    Apply `certificate.yaml` in `istio-system` namespace
+
+    ```
+    kubectl apply -f certificate.yaml -n istio-system
+    ```
+
+    After applying the above Certificate, cert-manager will generate the TLS certificate inside the istio-ingressgateway-certs secrets. The istio-ingressgateway-certs secret is mounted on the istio-ingressgateway deployment and used to serve HTTPS.
+
+1. [Configure Redirect URI for App](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app#add-a-redirect-uri)
+
+   Add below redirect uri to the app registered with Microsoft Identity:
+
+   `https://<LoadBalancer IP or DNS Name>/login/oidc`
+
+   **Note:** Make sure the redirect uri for the app matches the `oidc_redirect_uri` value in OIDC auth service settings.
+
+   Navigate to `https://<LoadBalancer IP or DNS Name>/` and start using Kubeflow.
+
+## Autheniticate Kubeflow pipelines using [Kubeflow Pipelines SDK](https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/)
+
+    {{< highlight python >}}
+    import kfp
```

Comment:
Is this highlight yams something website specific? It's not rendered good on GitHub